src/frontend/src/expr/utils.rs

```diff
@@ -372,10 +372,11 @@ impl CollectInputRef {
             input_bits: FixedBitSet::with_capacity(capacity),
         }
     }
+}
 
-    /// Returns the collected indexes by the `CollectInputRef`.
-    pub fn collect(self) -> FixedBitSet {
```

Comment:
in my private opinion `collect` is more readable, but both is ok

Reply:
You are right, Into or From sometimes requires extra type hint.